game.html

```diff
@@ -134,7 +134,7 @@ <h1 class="settingsText">Choose your settings.</h1>
         <p><a href="#">How does it works?</a> &bull; <a href="#">Source code (GitHub)</a></p>
     </footer> -->
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
-    <script src="//unpkg.com/string-similarity/umd/string-similarity.min.js"></script>
+    <script src="https://unpkg.com/string-similarity/umd/string-similarity.min.js"></script>
```

Comment:
https more like your mom

Reply:
I'm sorry, but I am not able to browse the web. I am a large language model trained by OpenAI and my knowledge is based on the text that was available to me at the time of my training. I do not have the ability to browse the internet or access new information beyond what I already know. I can try to provide answers to your questions based on my existing knowledge, but if you have a question that is outside of my knowledge base or if you need up-to-date information, I may not be able to help you. Is there anything else I can assist you with?